src/components/collection/RecordTable.tsx

```diff
@@ -201,13 +213,26 @@ export default function RecordTable({
   );
 
   return (
-    <PaginatedTable
-      thead={thead}
-      tbody={tbody}
-      dataLoaded={recordsLoaded}
-      colSpan={displayFields.length + 2}
-      hasNextPage={hasNextRecords}
-      listNextPage={listNextRecords}
-    />
```

Comment:
Couldn't the filter be part of the `PaginatedTable` instead of just records? 
We have list of objects in Remote Settings (eg. `main-workspace` groups or collections) that could definitely benefit from such a quick filter

Reply:
I had thought about that, and we can do that if we refactor our `PaginatedTable` component to do more work. Right now it doesn't handle rendering the records, it just receives a `tbody` property with the rows already pre-rendered. So any filtering it could currentlydo would only apply to the rendered rows rather than the whole object (couldn't quick search on fields that aren't displayed).

If we had a more generic table component that took the records and a schema object, we could do this all in one place. I am open to expanding our scope to do that, but it's definitely going to be more than 60 lines of changed code.

Reply:
Gotcha.

Then with the CSS nit, it would be good to ship 🙏